{
  "id": "EUVD-2018-19162",
  "enisaUuid": "51fa4358-9fa2-343b-bd44-6109ada64c44",
  "description": "Directory traversal vulnerability in the Splunk Django App in Splunk Enterprise 6.0.x before 6.0.14, 6.1.x before 6.1.13, 6.2.x before 6.2.14, 6.3.x before 6.3.10, 6.4.x before 6.4.6, and 6.5.x before 6.5.3; and Splunk Light before 6.6.0 allows remote authenticated users to read arbitrary files via unspecified vectors.",
  "datePublished": "Oct 23, 2018, 9:00:00 PM",
  "dateUpdated": "Aug 5, 2024, 6:24:12 AM",
  "baseScore": 6.5,
  "baseScoreVersion": "3.0",
  "baseScoreVector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N",
  "references": [
    "https://www.splunk.com/view/SP-CAAAP5T"
  ],
  "aliases": [
    "CVE-2018-7431",
    "GSD-2018-7431"
  ],
  "assigner": "mitre",
  "epss": 0.39,
  "enisaIdProduct": [
    {
      "id": "04475766-294c-3208-a96b-45138a2ecd16",
      "product": {
        "name": "n/a"
      },
      "product_version": "n/a"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "e9d609b5-0328-37cb-b7ba-68d05a31a8ca",
      "vendor": {
        "name": "n/a"
      }
    }
  ],
  "cve": "CVE-2018-7431"
}